doc/source/conf.py

```diff
@@ -17,6 +17,14 @@
 
 needs_sphinx = '1.6'
 
+# Workaround for sphinx-doc/sphinx#6573
+# ua._Function should not be treated as an attribute
+from sphinx.util import inspect
+import scipy._lib.uarray as ua
+old_isattrdesc = inspect.isattributedescriptor
+inspect.isattributedescriptor = (lambda obj: old_isattrdesc(obj)
+                                 and not isinstance(obj, ua._Function))
```

Comment:
It might be possible just to exclude based on the name:

https://github.com/scipy/scipy/blame/master/doc/source/_templates/autosummary/class.rst#L33

But might not be worth tinkering with further, I know Sphinx can be a pain and I'm happy enough that there is a suitable solution already...

Reply:
So if I understand correctly, I could edit the attribute template and add a special case for `ua._Function`?

I think I prefer the way it's done here because `conf.py` is a far more obvious place to look than in `_templates/autosummary/attribute.rst`. Especially since I'm trying to document a function.

Another solution I had considered was to use the environment variable to change my `_dispatch` decorator but I thought this was clearer.

Reply:
Okay let's keep as is